cirq/protocols/json_test.py

```diff
@@ -259,10 +259,6 @@ def test_fail_to_resolve():
     cirq.Z,
     'S':
     cirq.S,
-    'SerializableDevice':
-    cirq.google.SerializableDevice.from_proto(
-        proto=cirq.google.known_devices.FOXTAIL_PROTO,
-        gate_set=cirq.google.XMON),
```

Comment:
Why was this removed?

Reply:
Ironically, SerializableDevice cannot yet be serialized into JSON.  I'll add that functionality later.